http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/NodepoolComponent.scala

```diff
@@ -32,7 +32,6 @@ case class NodepoolRecord(
   creator: WorkbenchEmail,
   createdDate: Instant,
   destroyedDate: Instant,
-  dateAccessed: Instant,
```

Comment:
What do we gain by deleting this? It's one of the core AuditInfo fields. I understand we aren't setting the value now, but does that mean we will never need it?

Reply:
if we're not using it (even worse, having bad data for it), I'd prefer removing it to keep things simple. `nodepool` isn't a user facing object, so the likehood of us needing this is very low.

Reply:
How would the effort/usefulness compare if we added a line of code to initialize dateAccessed to dateCreated? My concern is that while the user can't see it, Leonardo can, and directly manages it; date of change can be important for traceability.

Reply:
If we want to properly use it, then we'd need update this whenever a user accesses an app on a nodepool, and it'll be different for GCP and Azure.

`NODEPOOL` table isn't useful for Azure apps at all, so I think future plan for this table is TBD as well.

Even if we can update it easily, what does that give us? in cases like runtimes, apps, we have it because we use it for autopause, but for nodepool, there's no use of it

Reply:
That's fair, and sounds like the removal will not have an impact.

Reply:
Would this be needed to track app stop/start once we fix it?

Reply:
nope...`dateAccessed` in `APP` table is what we need

Reply:
Eventually I hope we can drop the `NODEPOOL` table entirely ;-)